README.md

```diff
@@ -19,7 +19,7 @@ It also implements some new functionality like custom skins, and the ability to
     - [Download the latest release here.](https://github.com/jasminegamedev/Fuji/releases/latest)
     - Unzip the project.
     - Put your mods into the Mods folder that comes with Fuji.
-    - Open `Celeste64Launcher.exe` to start playing!
+    - Open `Celeste64.Launcher.exe` to start playing!
```

Comment:
Minor unimportant thing, but I might just change this later in a separate commit. I don't know if I really like having a period in the assembly name like this, so I might try to change the assembly name to something a little more user friendly.